spec/view_helper_spec.rb

```diff
@@ -51,6 +51,21 @@
         expect(meta).to have_tag('meta', with: { content: "facebook description", property: "og:description" })
       end
     end
+
+    it 'should work with app links parameters' do
+      subject.set_meta_tags('al' => {
+        'ios' => {
+          'url' => 'applinks://docs',
+          'app_store_id' => 12345,
+          'app_name' => 'App Links'
+        }
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.